project1-sps/script.js

```diff
@@ -1,4 +1,79 @@
+var NumPlayerWins = 0;
+var NumPlayerlosses = 0;
+var NumPlayerdraws = 0;
+
+var mode = "get user name";
+var UserName = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue;
+  if (mode == "get user name") {
+    //save username in global variable
+    UserName = input;
+    // change mode to play game
+    mode = "play game";
+    myOutputValue = `Hello ${UserName}`;
+  } else {
+    var playerInput = input;
+    var computerInput = getRandomObject();
+    console.log(computerInput);
```

Comment:
Nice console.logs for debugging - even better if you can label the console log (like computer input: ${computerInput}` so you know what you are logging